packages/@react-spectrum/tooltip/docs/Tooltip.mdx

```diff
@@ -0,0 +1,175 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/tooltip';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/tooltip/package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Edit from '@spectrum-icons/workflow/Edit';
+import {Flex} from '@react-spectrum/layout';
+import Question from '@spectrum-icons/workflow/Question';
+import Resize from '@spectrum-icons/workflow/Resize';
+import Save from '@spectrum-icons/workflow/SaveTo';
+import {Text} from '@react-spectrum/text';
+import ThumbUp from '@spectrum-icons/workflow/ThumbUp';
+import { Tooltip, TooltipTrigger } from '@react-spectrum/tooltip';
+```
+
+---
+category: Overlays
+keywords: [tooltip]
+---
+
+# Tooltip
+
+<p>{docs.exports.Tooltip.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Tooltip']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/tooltip/'}
+  ]} />
+
+## Examples
+Try hovering or focusing the button. On hover, it'll appear after a short delay. On focus it'll appear instantly. Clicking or hitting Escape will dismiss it.
+```tsx example
+<TooltipTrigger>
+  <ActionButton aria-label="Edit Name"><Edit /></ActionButton>
+  <Tooltip>Change Name</Tooltip>
+</TooltipTrigger>
+```
+
+### Delay
+The above mentioned delay after hover can be adjusted. Focus will always appear instantly. Setting it to 0 will cause hover to be instant as well.
+```tsx example
+<TooltipTrigger delay={0}>
+  <ActionButton aria-label="Save"><Save /></ActionButton>
+  <Tooltip>Saving applies your new settings right away.</Tooltip>
+</TooltipTrigger>
+```
+
+## Content
+
+Tooltips can have a label provided as a child. This can be an internationalized string.
+
+### Accessibility
+
+Tooltips will use their content to describe their trigger, they should not be complex. Tooltip triggers must be focusable and hoverable in order to ensure that all users can activate them.
```

Comment:
```suggestion
Tooltip triggers must be focusable and hoverable in order to ensure that all users can activate them. When displayed, TooltipTrigger automatically associates the tooltip with the trigger element so that it is described by the tooltip content.
```